preconditions-assertj/src/main/java/com/palantir/logsafe/testing/LoggableArgRepresentation.java

```diff
@@ -17,20 +17,25 @@
 package com.palantir.logsafe.testing;
 
 import com.palantir.logsafe.Arg;
+import org.assertj.core.presentation.Representation;
 import org.assertj.core.presentation.StandardRepresentation;
 
-public class LoggableArgRepresentation extends StandardRepresentation  {
+final class LoggableArgRepresentation extends StandardRepresentation {
```

Comment:
We can keep this public to avoid breaks, but we probably didn't want it to be public in the first place